spec/bolt/applicator_spec.rb

```diff
@@ -51,7 +51,7 @@
         config: {
           transport: 'ssh',
           transports: {
-            ssh: { 'connect-timeout' => 10, 'host-key-check' => true, tty: false },
+            ssh: { 'connect-timeout' => 10, 'host-key-check' => true, tty: false, 'load-config' => true },
```

Comment:
Why are we loading config for tests?

Reply:
This is the "expected" value for the input to catalog compilation. It expects `load-config => true` because that's the default.

Reply:
oh right. I remember hating this last time I had to change it.